guides/common/modules/con_prerequisites-for-network-boot-provisioning.adoc

```diff
@@ -3,6 +3,9 @@
 [id="prerequisites-for-network-boot-provisioning"]
 = Prerequisites for network-boot provisioning
 
+[role="_abstract"]
+Before you can use the automated PXE or UEFI HTTP provisioning methods, ensure that your environment meets the following requirements.
```

Comment:
```suggestion
Before you can perform network-based host provisioning, ensure that your environment meets the following requirements.
```

suggestion to avoid using those two specific provisioning methods.